game/configuration.cc

```diff
@@ -131,24 +131,27 @@ std::string const ConfigItem::getValue() const {
 		return "Auto";
 	}
 	if (this->getName() == "game/language") {
-		int autoLanguageType = 1337;
-		int val = LanguageToLanguageId(this->getEnumName()); // In the case of the language, val is the real value while m_value is the enum case for its cosmetic name.
+		unsigned autoLanguageType = 1337;
+		unsigned val = LanguageToLanguageId(this->getEnumName()); // In the case of the language, val is the real value while m_value is the enum case for its cosmetic name.
 		std::string languageName = (val != autoLanguageType) ? this->getEnumName() : "Auto";
 		return languageName;
 	}
 	if (m_type == "int") {
-		int val = std::get<int>(m_value);
-		if (val >= 0 && val < int(m_enums.size())) return m_enums[val];
-		return numericFormat<int>(m_value, m_multiplier, m_step) + _(m_unit);
+		unsigned val = static_cast<unsigned>(std::get<int32_t>(m_value));
```

Comment:
why not just get unsigned int 32?

Reply:
> why not just get unsigned int 32?

This was actually a bug, for a different reason.